Cargo.toml

```diff
@@ -25,11 +25,11 @@ smallvec = { version = "1.5.1", default-features = false }
 thiserror = "1.0.22"
 
 # used for tests and HTTP/hyper
-# the features are used only for tests but enabled because `dev-depedencies` are leaked into dependencies.
-tokio = { version = "0.2.24", features = ["stream", "rt-threaded", "macros"], optional = true }
+# the features are used only for tests but enabled because `dev-dependencies` are leaked into dependencies.
+tokio = { version = "1.0", features = ["net", "rt-multi-thread", "macros"], optional = true }
 
 # HTTP-related dependencies
-hyper = { version = "0.13.9", features = ["stream"], optional = true }
+hyper = { version = "0.14", features = ["stream", "client", "server", "http1", "http2", "tcp"], optional = true }
```

Comment:
note, we are only using `http1` currently but I think those deps are such small so that it doesn't matter.